tests/test_matplotlibFormatter.py

```diff
@@ -27,6 +27,8 @@
 import unittest
 from random import Random
 
+import numpy as np
```

Comment:
This needs to be a protected import since numpy is not butler requirement (although effectively it is since astropy won't work without it).